src/Common/src/Interop/Windows/user32/Interop.SetWindowLongPtr.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        [DllImport(Libraries.User32, CharSet = CharSet.Auto)]
+        public static extern IntPtr SetWindowLongPtr(IntPtr hWnd, int nIndex, IntPtr dwNewLong);
```

Comment:
This is not working on 32 bit -- this may help
https://stackoverflow.com/questions/3343724/how-do-i-pinvoke-to-getwindowlongptr-and-setwindowlongptr-on-32-bit-platforms

2018-02-23 22:26:05,404: INFO: proc(54): run_and_log_output: Output: Unhandled Exception: System.EntryPointNotFoundException: Unable to find an entry point named 'SetWindowLongPtr' in DLL 'user32.dll'.
2018-02-23 22:26:05,404: INFO: proc(54): run_and_log_output: Output:    at Interop.User32.SetWindowLongPtr(IntPtr hWnd, Int32 nIndex, IntPtr dwNewLong)

Reply:
I see, I removed some code around this because I thought it was OS version related and not architecture, I will add it back.